src/ansys_sphinx_theme/extension/linkcode.py

```diff
@@ -197,8 +197,6 @@ def link_code(app: Sphinx, doctree: Node):
     The `link_code_library` configuration option can be used to specify a custom library
     for link resolution.
 
-    Notes
-    -----
     Adapted from the original `linkcode` extension in Sphinx, see
```

Comment:
```suggestion
    Adapted from the original ``linkcode`` extension in Sphinx. For more information, see
```
What is adapted? This sentence needs a sentence and it should have display text for the link (rather than the URL itself).

Reply:
changed this to
```
The ``linkcode`` extension in this version has been adapted from the original 
`Sphinx linkcode extension <https://github.com/sphinx-doc/sphinx/blob/main/sphinx/ext/linkcode.py>`_.
This adaptation involves modifying the original code and functionality from the Sphinx library.
```